apps/prairielearn/src/pages/administratorWorkspaces/administratorWorkspaces.html.ts

```diff
@@ -109,46 +86,58 @@ function AdministratorWorkspaces({ workspaces, workspaceLoadHostCapacity, resLoc
-                      <div class="list-group list-group my-2">
-                        ${workspaceHost.workspaces.map((workspace) => {
-                          const maybeCourseInstanceName = workspace.course_instance_name
-                            ? html`(<span title="Course instance"
-                                  >${workspace.course_instance_name}</span
-                                >)`
-                            : null;
-                          return html`
-                            <div class="list-group-item">
-                              <div class="d-flex align-items-center">
-                                <span class="mr-2" style="font-variant-numeric: tabular-nums;">
-                                  ${workspace.id}
-                                </span>
-                                <span> </span>
-                                ${WorkspaceState({ state: workspace.state })}
-                                <span class="badge badge-secondary">
-                                  ${workspace.time_in_state}
-                                </span>
-                              </div>
-                              <div class="text-muted text-small">
-                                <span class="text-monospace" title="Question"
-                                  >${workspace.question_name}</span
-                                >
-                                &bull;
-                                <span title="Course">${workspace.course_name}</span>
-                                ${maybeCourseInstanceName} &bull;
-                                <span title="Institution">${workspace.institution_name}</span>
-                              </div>
+                      ${workspaces.length === 0
+                        ? html`
+                            <div class="text-muted my-2">
+                              There are no workspaces running on this host.
                             </div>
-                          `;
-                        })}
-                      </div>
+                          `
+                        : html`
+                            <div class="list-group list-group my-2">
+                              ${workspaces.map((workspace) => {
+                                const maybeCourseInstanceName = workspace.course_instance_name
+                                  ? html`(<span title="Course instance"
+                                        >${workspace.course_instance_name}</span
+                                      >)`
+                                  : null;
+                                return html`
+                                  <div class="list-group-item">
+                                    <div class="d-flex align-items-center">
+                                      <span
+                                        class="mr-2"
+                                        style="font-variant-numeric: tabular-nums;"
```

Comment:
I know this was already in the file before, but this styling seems a bit unconventional, particularly for an ID that doesn't actually have a numeric significance. Is there a reason for it? And would it make sense to create a CSS class for it?

Reply:
It makes things line up a bit more nicely, that's all. I don't think there's a need for a class yet; maybe if we start using this more widely?